crates/volta-layout/src/v1.rs

```diff
@@ -0,0 +1,10 @@
+use volta_layout_macro::layout;
+
+pub use super::v0::VoltaHome;
```

Comment:
This may just be my not following how the macro works (because I haven't dug into it at all), but what is this `pub use` here for?

Reply:
Since at this stage there aren't any changes to the `VoltaHome` part of the layout, this is re-exporting the `v0` Layout to be part of `v1`. That lets us use `v1::VoltaHome` and `v1::VoltaInstall` as the two layout structs everywhere, instead of having a mixed state of `v0::VoltaHome` and `v1::VoltaInstall`

This also changes in one of the later PRs, once changes are actually made to the layout, so this is replaced by the actual `VoltaHome` struct.

Reply:
Ahhhh, got it. Thanks for clarifying – I was reading that very PR so it makes sense!